### ISSUE TITLE
Is it possible that the number of keypoints in every samples is not same?

### ISSUE THREAD

**Issue by yulong314** [open-mmlab/mmpose#394]
As far as I know that the keypoint recognition methods such as mmpose require that  every sample have the same number of keypoints labelled.
 What if I have samples that every one of them only labelled a portion of all keypoints.
 Do I need to modify the loss function such that  only keypoints that have ground truth give loss values, other keypoints do not give loss values?

**Comment by jin-s13** [comment]
That's fine. 
Just leave these unlabelled keypoints as [0, 0, 0] and set `use_target_weight=True` (which is our default settings). 
During training, these keypoints will be ignored and no loss will be back-propagated.

**Comment by yulong314** [comment]
> That's fine.
> Just leave these unlabelled keypoints as [0, 0, 0] and set `use_target_weight=True` (which is our default settings).
> During training, these keypoints will be ignored and no loss will be back-propagated.

Thanks a lot for your answer. That's really great!
